### ISSUE TITLE
CJB Cheats Menu 1.20.1

### ISSUE THREAD

**Issue by Pathoschild** [StardewModders/mod-translations#16]
**[CJB Cheats Menu](https://www.nexusmods.com/stardewvalley/mods/4)** adds an in-game cheats menu with lots of cheats.

## Requested translations
See [default text](https://github.com/CJBok/SDV-Mods/blob/develop/CJBCheatsMenu/i18n/default.json) and [translation guide](https://stardewvalleywiki.com/Modding:Translations). These translations are requested (see lines marked `// TODO`):

language | translations | status
-------- | ------------ | ------
~Chinese~ | ~[zh.json](https://github.com/CJBok/SDV-Mods/blob/develop/CJBCheatsMenu/i18n/zh.json)~ | [updated](https://github.com/CJBok/SDV-Mods/pull/84) (thanks to @Redlnn!)
French | [fr.json](https://github.com/CJBok/SDV-Mods/blob/develop/CJBCheatsMenu/i18n/fr.json) | incomplete (see lines marked `// TODO`)
~German~ | ~[de.json](https://github.com/CJBok/SDV-Mods/blob/develop/CJBCheatsMenu/i18n/de.json)~ | [updated](https://github.com/CJBok/SDV-Mods/pull/77) (thanks to @jahangmar!)
Hungarian | | missing
Italian | | missing
~Korean~ | ~[ko.json](https://github.com/CJBok/SDV-Mods/blob/develop/CJBCheatsMenu/i18n/ko.json)~ | updated (thanks to @overwritten!)
~Portuguese~ | ~[pt.json](https://github.com/CJBok/SDV-Mods/blob/develop/CJBCheatsMenu/i18n/pt.json)~ | [updated](https://github.com/CJBok/SDV-Mods/pull/80) (thanks to @shiro2579!)
Russian | [ru.json](https://github.com/CJBok/SDV-Mods/blob/develop/CJBCheatsMenu/i18n/ru.json) | incomplete (see lines marked `// TODO`)
~Spanish~ | ~[es.json](https://github.com/CJBok/SDV-Mods/blob/develop/CJBCheatsMenu/i18n/es.json)~ | [updated](https://github.com/CJBok/SDV-Mods/pull/79) (thanks to @Riazaia!)
Turkish | | missing

Most translations appear in the menu shown when you press `P` in-game. For example:

* `tabs.quests`, `activequests.title`, and `completedquests.title` are shown on the Quests tab:  
  > ![](https://user-images.githubusercontent.com/230581/50747758-a42b1700-1203-11e9-899a-d384d6438cbb.png)

* `fast-machines.fruit-trees` is shown on the Farm & Fishing tab:  
  > ![](https://user-images.githubusercontent.com/230581/50747789-cf156b00-1203-11e9-9741-b23cb21a5158.png)

* `messages.press-new-key` is shown on the Controls tab when you click 'set':  
  > ![](https://user-images.githubusercontent.com/230581/50747816-fc621900-1203-11e9-98ef-9d8c12eca5d4.png)

This one appears outside the menu:
* `messages.time-frozen` is shown in the top-left corner when you freeze time (press `T` by default):
  > ![](https://user-images.githubusercontent.com/230581/50747835-1ac81480-1204-11e9-8a14-c9e10ff28049.png)

## Credit
Full credit for translators is provided in the release notes. Thanks!

## See also
* [source code](https://github.com/CJBok/SDV-Mods/tree/develop/CJBCheatsMenu)

**Comment by overwritten** [comment]
here's some translations for the ko.json file! the advanced tab is assuming that "advanced settings" is implied, but the word itself just says "advanced" ^___^

```
"tabs.advanced": "고급",

"flags.warning": "이 절에서는 당신 자신의 위험을 감수하고 사용하세요!\n따라서 메일 또는 이벤트나 퀘스트 생력하는데 문제가 발생할수 있어요.",
"flags.quests": "완료한 퀘스트",
"flags.unlocked.guild": "모험가 길드",
"flags.unlocked.room": "방: {{name}}",
"flags.community-center": "마을 회관",
"flags.community-center.door-unlocked": "문을 열렸기",
"flags.jojamart.membership": "조자마트 회원",
```

**Comment by Pathoschild** [comment]
@overwritten Thanks a lot! Added in the next CJB Cheats Menu update.

**Comment by overwritten** [comment]
> @overwritten Thanks a lot! Added in the next CJB Cheats Menu update.

There's still two lines missing in the flags menu! I just got them checked and translated:

```
"flags.wallet": "지갑 아이템들",
"flags.unlocked": "해금된 지역들",
```

**Comment by Pathoschild** [comment]
Thanks again!

**Comment by jahangmar** [comment]
german translation is [done](https://github.com/CJBok/SDV-Mods/pull/77) but the following line has a typo:
`    "flags.community-center.door-unlocked": "Tür aufgeschlossen",` is correct.

**Comment by Pathoschild** [comment]
Fixed, thanks!

**Comment by Pathoschild** [comment]
CJB Cheats Menu was updated with new translations; closed in favor of #20.
